src/pages/Lending/AvailablePools.tsx

```diff
@@ -65,6 +69,15 @@ const LendingPoolButton = ({ asset, onPoolClick }: LendingPoolButtonProps) => {
   )
 
   const StatusTag = useCallback(() => {
+    if (!isThorchainLendingBorrowEnabled && !isThorchainLendingRepayEnabled) {
```

Comment:
blocking: this one should absolutely use the `PAUSELOANS` mimir.

i.e only display me if *explicitly* paused at mimir-level. If *we* disabled it, we should either:
- display some "Disabled" copy of sorts or
- do nothing and keep on handling the way we currently do

Reply:
Done in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)